src/wp-includes/blocks.php

```diff
@@ -73,6 +73,52 @@ function generate_block_asset_handle( $block_name, $field_name, $index = 0 ) {
 	return $asset_handle;
 }
 
+/**
+ * Get url to block assets.
+ *
+ * @since 6.3.0
+ * @param string $path Path to assert.
+ * @param string $file Path to block.json file.
+ *
+ * @return string|null
+ */
+function get_block_asset_uri( $path, $file ) {
```

Comment:
The docblock and the function signature use URL and URI interchangeably. Given that we're always returning a URL (which is a type of URI), I'd suggest consolidating on URL. Also leaving additional docs suggestions below.

1. "URL" should always be capitalized since it's an acronym
2. Clarify param docs and add a return description
3. Remove `null` return type, as I don't see where this could return a `null` value.
4. Update the function name to use `_url` instead of `_uri` (will require changes throughout where this is used)

```suggestion
/**
 * Get the URL to a block asset.
 *
 * @since 6.4.0
 * @param string $path A normalized path to a block asset.
 * @param string $file The path to the block.json file.
 *
 * @return string The URL to the block asset.
 */
function get_block_asset_url( $path, $file ) {
```

As an aside, is there any value in this function for general use, or should we prefix this with an underscore and mark as private?